### PR TITLE
update sliding window clustering FCCee with k4geo

### PR DESCRIPTION
**Pull request by dasphy**
Hello @BrieucF , @kjvbrt
PR #16 is discarded, this PR is the replacement.

@giovannimarchiori 
@gartrog 

Tong

### CODE REVIEW

**Comment by BrieucF** [comment]
Thanks @dasphy , I'll merge after https://github.com/HEP-FCC/k4RecCalorimeter/pull/58 and 
https://github.com/key4hep/k4FWCore/pull/158

File: FCCSW_ecal/run_thetamodulemerged.py [hunk 1]
```diff
@@ -4,7 +4,9 @@
 from Configurables import PodioOutput
 from Configurables import CorrectCaloClusters
 from Configurables import CreateCaloClustersSlidingWindow
```
Comment (giovannimarchiori) [inline FCCSW_ecal/run_thetamodulemerged.py:6]: if CreateCaloClustersSlidingWindow is not used anymore, can you remove this line?

File: FCCSW_ecal/run_thetamodulemerged.py [hunk 2]
```diff
@@ -4,7 +4,9 @@
...
+from Configurables import CreateCaloClustersSlidingWindowFCCee
 from Configurables import CaloTowerTool
```
Comment (giovannimarchiori) [inline FCCSW_ecal/run_thetamodulemerged.py:8]: if CaloTowerTool is not used anymore, can you remove this line?

File: FCCSW_ecal/run_thetamodulemerged.py [hunk 3]
```diff
@@ -414,7 +414,8 @@
                        hcalEndcapReadoutName="",
                        hcalFwdReadoutName="",
                        OutputLevel=INFO)
-towers.ecalBarrelCells.Path = ecalBarrelCellsName
+towers.ecalBarrelCells.Path = ecalBarrelPositionedCellsName
+#towers.ecalBarrelCells.Path = ecalBarrelCellsName
```
Comment (giovannimarchiori) [inline FCCSW_ecal/run_thetamodulemerged.py:418]: can you remove this commented line?

Comment (dasphy) [reply FCCSW_ecal/run_thetamodulemerged.py:418]: ok removing these

**Comment by dasphy** [comment]
> Thanks @dasphy , I'll merge after [HEP-FCC/k4RecCalorimeter#58](https://github.com/HEP-FCC/k4RecCalorimeter/pull/58) and [key4hep/k4FWCore#158](https://github.com/key4hep/k4FWCore/pull/158)

Hi @BrieucF 
The 2 PRs above are just merged.
Cheers, Tong

Tagging: @giovannimarchiori , @gartrog

**Comment by BrieucF** [comment]
Great! It looks good to me. Any further comment @giovannimarchiori ?

@dasphy can you also port this to https://github.com/HEP-FCC/FCC-config/blob/main/FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v02/run_digi_reco.py ? This repository will host clean configurations for the digitization and reconstruction of the full FCC detectors concepts, in view of future central productions.

**Comment by dasphy** [comment]
> @dasphy can you also port this to https://github.com/HEP-FCC/FCC-config/blob/main/FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v02/run_digi_reco.py ? This repository will host clean configurations for the digitization and reconstruction of the full FCC detectors concepts, in view of future central productions.

OK no problem

**Comment by giovannimarchiori** [comment]
> Great! It looks good to me. Any further comment @giovannimarchiori ?
> 
> @dasphy can you also port this to https://github.com/HEP-FCC/FCC-config/blob/main/FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v02/run_digi_reco.py ? This repository will host clean configurations for the digitization and reconstruction of the full FCC detectors concepts, in view of future central productions.

Looks good to me, thanks!

**Pull request merged**